test/metabase/models/setting_test.clj

```diff
@@ -266,69 +266,71 @@
 
 (deftest admin-writable-settings-test
   (testing `setting/admin-writable-settings
-    (test-setting-1! nil)
-    (test-setting-2! "TOUCANS")
-    (is (= {:key            :test-setting-2
-            :value          "TOUCANS"
-            :description    "Test setting - this only shows up in dev (2)"
-            :is_env_setting false
-            :env_name       "MB_TEST_SETTING_2"
-            :default        "[Default Value]"}
-           (some (fn [setting]
-                   (when (re-find #"^test-setting-2$" (name (:key setting)))
-                     setting))
-                 (setting/admin-writable-settings))))
-
-    (testing "with a custom getter"
+    (mt/with-test-user :crowberto
```

Comment:
The diff here isn't super clear but I only added this line (to keep the test functioning); the rest is the same just shifted down.